README

```diff
@@ -9,29 +9,70 @@ Directory contents:
 		target built from this directory is libest.a.
     /example	This directory contains sample code for implementing
                 an EST client or server.
-    /test	This directory contains unit test code to verify
-                the libest capabilities.
-
 
 Getting started
 ---------------
-1.  ./configure; make; make install
```

Comment:
Not sure what this is also missing in the internal version, but the statement that "make install" be performed should not be getting removed.